packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -593,8 +596,10 @@ async function handler(req: CustomRequest) {
     }
 
     // Posible to refactor TODO:
-    const paymentApp = await appStore[paymentAppCredential?.app?.dirName as keyof typeof appStore]();
-    if (!(paymentApp && "lib" in paymentApp && "PaymentService" in paymentApp.lib)) {
+    const paymentApp = (await appStore[
+      paymentAppCredential?.app?.dirName as keyof typeof appStore
+    ]()) as PaymentApp;
+    if (!paymentApp.lib?.PaymentService) {
```

Comment:
Why are we doing a type assertion? Earlier code was safer as it's possible that paymentApp or paymentApp.lib is nullish

Reply:
I was getting a weird type error in this file. Here's a Loom with what I was experiencing. Still the same app.

https://www.loom.com/share/7dd62f03e6af463bb51605408b0a9e8c